docs/index.md

```diff
@@ -44,84 +44,46 @@
-video](https://www.youtube.com/watch?v=R2nr1uZ8ffc&list=PLgkF0qak9G49QlteBtxUIPapT8TzfPuB8)
-recorded at our PyData talk on December 2019. In it we describe the motivation
-behind Streamlit, then go over how to install and create apps with it.
+   `pip install streamlit`
 
-## Install Streamlit
+3. Run the hello world app:
 
-```eval_rst
-.. raw:: html
-
-  <div class='embed-container'><iframe
-    width="560"
-    height="315"
-    src="https://www.youtube.com/embed/sxLNCDnqyFc"
-    style="margin: 0 0 2rem 0;"
-    frameborder="0"
-    allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
-    allowfullscreen>
-  </iframe></div>
-```
+   `streamlit hello`
 
-Follow these steps and to get a sample app running in less than 5 minutes.
+4. That's it! In the next few seconds the sample app will open in a new tab in your default browser.
 
-1. Make sure that you have [Python 3.6](https://www.python.org/downloads/) or greater installed.
-1. Install Streamlit using [PIP](https://pip.pypa.io/en/stable/installing/):
+Still with us? Great! Now make your own app in just 3 more steps:
 
-   ```bash
-   pip install streamlit
-   ```
+1. Open a new Python file, import Streamlit, and write some code
 
-1. Run the hello world app:
+2. Run the file with:
 
-   ```bash
-   streamlit hello
-   ```
+   `streamlit run [filename]`
 
-1. That's it! In the next few seconds the sample app will open in a new tab in
-   your default browser.
+3. When you're ready, click 'Deploy' from the Streamlit menu and use Streamlit sharing to share your app with the world!
 
-## Get started
+Now that you're set up, let's dive into more of how Streamlit works and how to build great apps.
 
-The easiest way to learn how to use Streamlit is to actually try it out. Our
-[get started guide](getting_started.md) walks you through the basics
-of building a Streamlit app.
+## How to use our docs
 
-## Build your first app
+The docs are broken up into 5 sections that will help you get the most out of Streamlit.
 
-```eval_rst
-.. raw:: html
-
-  <div class='embed-container'><iframe
-    width="560"
-    height="315"
-    src="https://www.youtube.com/embed/VtrFjkSGgKM"
-    style="margin: 0 0 2rem 0;"
-    frameborder="0"
-    allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
-    allowfullscreen>
-  </iframe></div>
-```
+**Tutorials**: include our [Get Started](getting_started.md) guide and a few step-by-step examples to building different types of apps in Streamlit.
+
+**Topic guides**: give you background on how different parts of Streamlit work. Make sure to check out the sections on [Creating an app](main_concepts.md) and [Deploying an app](deploy_streamlit_app.md), and for you advanced users who want to level up your apps, be sure to read up on [Caching](caching.md) and [Components](develop_streamlit_components.md).
+
+**Cookbook**: provides short code snippets that you can copy in for specific use cases.
```

Comment:
Docs say 5, but there's only 4 sections on the right side; Cookbook is integrated into Topic Guides?

Reply:
Yeah, this is a formatting issue I just won't have time for before release. Cookbook should be separate as a section